ecommerce/src/main/kotlin/io/hhplus/ecommerce/product/application/ProductStatisticsScheduler.kt

```
@@ -0,0 +1,239 @@
+                    statistics.addSalesCount(aggregatedStats.salesCount)
+                }
+                if (aggregatedStats.wishCount != 0L) {
+                    statistics.updateWishCount(aggregatedStats.wishCount)
+                }
+
+                productPermanentStatisticsRepository.save(statistics)
+            }
+
+            logger.debug("[이벤트배치] 청크 업데이트 완료: ${chunk.size}건")
+
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)